src/profile_manager/templates/profile_manager/profile_detail.html

```diff
@@ -103,7 +103,9 @@ <h3 class="panel-title">General</h3>
       </div>
       <div class="panel-body">
         <p>Username: {{ object.user.get_username}}</p>
-        <p>Grad Year: {% if object.grad_year %}{{ object.grad_year}}{% else %}<i>None. Edit your profile!</i>{% endif %}</p>
+        {% if config.custom_profile_field %}
+        <p>{{ config.custom_profile_field }}: {% if object.custom_profile_field %}{{ object.custom_profile_field}}{% else %}<i>None.</i>{% endif %}</p>
```

Comment:
Django has a template filter to make this logic cleaner and avoid the if statements:
https://docs.djangoproject.com/en/5.0/ref/templates/builtins/#default-if-none

Reply:
Also: https://docs.djangoproject.com/en/5.0/ref/templates/builtins/#default

Reply:
added